torchvision/prototype/datasets/_builtin/README.md

```diff
@@ -12,51 +12,66 @@ Finally, `from torchvision.prototype import datasets` is implied below.
 
 Before we start with the actual implementation, you should create a module in `torchvision/prototype/datasets/_builtin`
 that hints at the dataset you are going to add. For example `caltech.py` for `caltech101` and `caltech256`. In that
-module create a class that inherits from `datasets.utils.Dataset` and overwrites at minimum three methods that will be
-discussed in detail below:
+module create a class that inherits from `datasets.utils.Dataset` and overwrites four methods that will be discussed in
+detail below:
 
 ```python
-from typing import Any, Dict, List
+import pathlib
+from typing import Any, BinaryIO, Dict, List, Tuple, Union
 
 from torchdata.datapipes.iter import IterDataPipe
-from torchvision.prototype.datasets.utils import Dataset, DatasetInfo, DatasetConfig, OnlineResource
+from torchvision.prototype.datasets.utils import Dataset, OnlineResource
 
+from .._api import register_dataset, register_info
+
+NAME = "my-dataset"
+
+@register_info(NAME)
+def info() -> Dict[str, Any]:
```

Comment:
They're already private, but for consistency, since we wrote all of these as `_info()`

```suggestion
def _info() -> Dict[str, Any]:
```